osmaxx/api_client/conversion_api_client.py

```diff
@@ -129,6 +129,18 @@ def _create_job_TODO_replace_me(self, extraction_order, request):  # noqa
             logging.error('Could not retrieve api job id from response.', response)
         return response
 
+    def get_result_file_url(self, job_id):
+        job_detail_url = CONVERSION_JOB_URL + '{}/'.format(job_id)
+        return self.authorized_get(job_detail_url).json()['resulting_file']
+
+    def download_file_to(self, url, destination_filename):
+        r = self.authorized_get(url, stream=True)
+        with open(destination_filename, 'wb') as f:
+            for chunk in r.iter_content(chunk_size=1024):
+                if chunk:
+                    f.write(chunk)
```

Comment:
## Chunking
Our files can be quite big, serving them in one load (loading them into memory) isn't a good idea. A much better solution would be to use NGINX's `X-SENDFILE`; something like [django-sendfile](https://pypi.python.org/pypi/django-sendfile) or [django-downloadview](https://django-downloadview.readthedocs.io/en/1.9/), see also an [older blogpost](http://www.sensibledevelopment.com/2010/11/django-sendfile-an-for-abstraction-large-file-serving-in-django/)

## Saving the file "again"

That's how this [must](http://stackoverflow.com/questions/7514964/django-how-to-create-a-file-and-save-it-to-a-models-filefield) be, unfortunately.

See also [FieldFile](https://docs.djangoproject.com/en/1.9/ref/models/fields/#django.db.models.fields.files.FieldFile.save) and [FileField](https://docs.djangoproject.com/en/1.9/ref/models/fields/#filefield-and-fieldfile)